hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedRocksObjectUtils.java

```diff
@@ -41,6 +41,20 @@ static void assertClosed(RocksObject rocksObject) {
     }
   }
 
+  static void assertClosed(ManagedObject<?> object) {
+    RocksObject rocksObject = object.get();
+    ManagedRocksObjectMetrics.INSTANCE.increaseManagedObject();
+    if (rocksObject.isOwningHandle()) {
+      ManagedRocksObjectMetrics.INSTANCE.increaseLeakObject();
+      LOG.warn("{} is not closed properly",
+          rocksObject.getClass().getSimpleName());
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("StackTrace for unclosed instance: {}",
+            object.getStackTrace());
+      }
+    }
```

Comment:
Would be nice to avoid duplicating logic from `assertClosed(RocksObject, Throwable)`.

Reply:
Done. There were no callers for `assertClosed(RocksObject, Throwable)`, removed it, maybe we can add later if it's of use.

Reply:
Thanks @sadanand48 for updating the patch.